css/s-forms.min.css

```diff
@@ -1 +1,96 @@
-.checkbox label{font-weight:700}.panel-title .checkbox label{font-weight:400}.answerable-question{color:#31708f;background-color:#d9edf7;border-color:#bce8f1;border-bottom:1px solid transparent;border-top-right-radius:3px;border-top-left-radius:3px;padding:0 15px}.answerable-subquestions{padding-top:10px;padding-left:2em;border:1px solid #bce8f1;border-bottom-left-radius:4px;border-bottom-right-radius:4px;margin-bottom:18px}svg{fill:currentColor}.caret-square-down,.caret-square-up,.info-circle,.question-circle{height:1rem;margin-bottom:.15rem}.caret-square-down,.caret-square-up{margin-right:.2rem}.help-icon-text-input{margin-bottom:25px;bottom:0}.has-unit-label{margin-bottom:26px;bottom:0;font-weight:700;position:absolute}.tooltip-content{white-space:pre-wrap}.collapse-toggle{margin-right:.3em;line-height:.9}.form-label{font-weight:500}input:disabled{cursor:not-allowed}.cursor-pointer{cursor:pointer}.accordion>.card{overflow:visible}.react-datepicker-wrapper{display:block}.react-datepicker__navigation:focus{outline:0}.wizard-step{min-height:650px}.wizard-step-content{min-height:600px}.list-group-item.active{font-weight:500}.question-category-1{background-color:rgba(8,255,0,.05)}.question-category-2{background-color:rgba(230,127,0,.05)}.question-category-3{background-color:rgba(222,0,230,.05)}.question-category-4{background-color:rgba(0,84,230,.05)}.question-category-5{background-color:rgba(0,230,222,.05)}
\ No newline at end of file
+.checkbox label {
```

Comment:
Not sure what is goal of that file anymore, do we use it in the app ? How it is created ?

Could you have a look if this file is particularly used in SForms-smart-components or is it different one ? 

The name suggest it should be mimi-fied version thus most-likely it makes sense just keep it as it was and just put it into ignore list of pretifier.

Reply:
I am not sure why there is this file, but yes it is supposed to be minified so it should be in prettierignore if we keep it. However, I looked on both s-forms and s-forms smart components and could see no difference. Minified css files are usually created at build time from the normal css file.

Here is the initial commit message:  

> Martin Ledvinka 8/29/16, 5:50 PM Added help icon for questions (based on dc:description property). Moved CSS file into a separate top-level folder, so that it can be copied into target projects. Bumped version to 0.0.8.

I guess it is not relevant anymore.

Reply:
It is used in s-forms-smart-components as follows:

In `ExampleApp.jsx`:
```
import 's-forms/css/s-forms.min.css';
```

Thus we need it. I suggest to ignore pretify and consider it resolved for now. We will tackle this by changing bundling library which will come next.

Reply:
After that i consider this issue resolved and can be merged to the master.

Reply:
Ok, I added prettier ignore file